test/e2e/framework/metrics_util.go

```diff
@@ -543,6 +543,7 @@ func sendRestRequestToScheduler(c clientset.Interface, op string) (string, error
 func getSchedulingLatency(c clientset.Interface) (*SchedulingMetrics, error) {
 	result := SchedulingMetrics{}
 	data, err := sendRestRequestToScheduler(c, "GET")
+	ExpectNoError(err)
```

Comment:
I don't think you should be triggering failure here. It's better to instead `return &result, err`.

Reply:
So that the error can be handled by the caller of this function.